indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,340 @@
+{% macro render_program(event, days, config) %}
+    {% for day in days %}
+        <div class="day pagebreak program-{{loop.index}} {{ 'pagebreak-children' if config.new_page_per_session else '' }}" id="day-{{ loop.index }}">
+            <div class="day-title">
+                <h2>{{ day|format_skeleton('EEEEdMMMM', timezone=event.timezone) }}</h2>
+            </div>
+            {% for entry in days[day] %}
```

Comment:
```suggestion
            {% for entry in entries %}
```

Reply:
We extract this but instead I can set a variable for it as such:

```suggestion
            {% set entries = days[day] %}
            {% for entry in entries %}
```